nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/configuration/Checker.java

```diff
@@ -33,8 +34,9 @@ and Distribution License("CDDL") (collectively, the "License").  You
     String getEnabled();
     void setEnabled(String value) throws PropertyVetoException;
 
-    @Attribute(defaultValue = "5")
-    Long getTime();
+    @Attribute(defaultValue = "5", dataType = Long.class)
+    @Min(value = 0)
```

Comment:
@mulderbaba Is this correct? I haven't spotted anything that indicates that it can't be less than 0